src/state/slices/common-selectors.ts

```diff
@@ -296,8 +296,10 @@ export const selectAssetsBySearchQuery = createCachedSelector(
 
     // Filters by low market-cap to avoid spew
     const filteredAssets = sortedAssets.filter(asset => {
+      // We don't have the market cap for TCY, but we want to show it
+      if (asset.assetId === tcyAssetId) return true
       const marketCap = marketDataUsd[asset.assetId]?.marketCap
-      return !marketCap || bnOrZero(marketCap).gte(1000)
+      return !marketCap || bnOrZero(marketCap).gte(10000)
```

Comment:
q: wondering if a fix here couldn't be to `bnOrZero(marketCap).isZero()` instead of `!marketCap`? That would generalise more.

<img width="1434" height="819" alt="Image" src="https://github.com/user-attachments/assets/6b7f6935-ccc1-423c-b1e2-87c64ff5c9e1" />

Did give this a quick spin and that seem to perform as expected too:
https://jam.dev/c/7c35d701-728d-411e-8d38-1f0da1237d27